linker/src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -77,4 +88,59 @@ class RealtimeLinkingController(
 
         return matchedEntityPairs
     }
+
+    @RequestMapping(
+            path = [RealtimeLinkingApi.BLOCKING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun block(
+            @RequestBody blockingRequest: BlockingRequest
+    ): Map<UUID, Map<UUID, List<BlockedEntity>>> {
+        ensureAdminAccess()
+        val propertyFqnCache = CacheBuilder
```

Comment:
this cache looks like it might be meant to live at the top level of the class (instead of being constructed inside this function)

Reply:
Good point, I put in locally in the function instead of adding a near cache on linker, because I didn't want to think through consequences of edm changes at this time.